src/ngx_http_lua_kong_module.c

```diff
@@ -48,11 +49,23 @@ static ngx_http_module_t ngx_http_lua_kong_module_ctx = {
     NULL                                     /* merge location configuration */
 };
 
+static ngx_command_t ngx_http_lua_kong_cmds[] = {
+
+    { ngx_string("lua_kong_load_var_index"),
+      NGX_HTTP_MAIN_CONF|NGX_CONF_TAKE1,
+      ngx_http_lua_kong_load_var_index,
+      NGX_HTTP_LOC_CONF_OFFSET,
```

Comment:
```suggestion
      0,
```

Since there is no `conf` associated with this directive, it does not need a valid offset in this case.